### ISSUE TITLE
 Error: An unexpected error occurred [ElasticsearchDomain]

### ISSUE THREAD

**Issue by b4nst** [infracost/infracost#522]
```
Detected Terraform directory at .
  ✔ Running terraform plan
  ✔ Running terraform show

Error: An unexpected error occurred

runtime error: index out of range [0] with length 0
goroutine 1 [running]:
runtime/debug.Stack(0xc0002b4450, 0xc0004b0300, 0x2d5)
        /opt/local/lib/go/src/runtime/debug/stack.go:24 +0x9f
main.handleUnexpectedErr(0xc0000f88f0, 0x16ee4a0, 0xc000298120)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/cmd/infracost/main.go:220 +0x45
main.main.func1(0xc0000e9ec8, 0xc0000f88f0, 0xc0000323c0)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/cmd/infracost/main.go:39 +0xa5
panic(0x16ee4a0, 0xc000298120)
        /opt/local/lib/go/src/runtime/panic.go:965 +0x1b9
github.com/infracost/infracost/internal/providers/terraform/aws.NewElasticsearchDomain(0xc0002b6280, 0x0, 0xc0002bae2c)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/internal/providers/terraform/aws/elasticsearch_domain.go:29 +0x37f6
github.com/infracost/infracost/internal/providers/terraform.(*Parser).createResource(0xc0000e9720, 0xc0002b6280, 0x0, 0x22)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/internal/providers/terraform/parser.go:65 +0x174
github.com/infracost/infracost/internal/providers/terraform.(*Parser).parseJSONResources(0xc0000e9720, 0xc00, 0x1bbc728, 0x0, 0x0, 0xc0002112c0, 0x5, 0xc0002bad80, 0xcaf, 0x0, ...)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/internal/providers/terraform/parser.go:102 +0x4d1
github.com/infracost/infracost/internal/providers/terraform.(*Parser).parseJSON(0xc0000e9720, 0xc0002ba000, 0xcaf, 0xcaf, 0xc0002112c0, 0x0, 0xc0000e9728, 0x14f8406, 0xc0002112c0, 0x0, ...)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/internal/providers/terraform/parser.go:123 +0x892
github.com/infracost/infracost/internal/providers/terraform.(*DirProvider).LoadResources(0xc0000f4630, 0xc0002112c0, 0xc0002112c0, 0x0, 0x0)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/internal/providers/terraform/dir_provider.go:104 +0x145
main.runMain(0xc000258000, 0xc0000f88f0, 0x0, 0x0)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/cmd/infracost/run.go:74 +0x388
main.breakdownCmd.func1(0xc000258000, 0xc000099400, 0x0, 0x2, 0x0, 0x0)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/cmd/infracost/breakdown.go:38 +0xdf
github.com/spf13/cobra.(*Command).execute(0xc000258000, 0xc0000993e0, 0x2, 0x2, 0xc000258000, 0xc0000993e0)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/spf13/cobra/command.go:852 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc00021d680, 0x351, 0xc0000e9ee8, 0x7)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/spf13/cobra/command.go:960 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/spf13/cobra/command.go:897
main.main()
        /opt/local/var/macports/build/_opt_bblocal_var_buildworker_ports_build_ports_sysutils_infracost/infracost/work/gopath/src/github.com/infracost/infracost/cmd/infracost/main.go:171 +0xd3e

Environment:
Infracost v0.8.1
```

**Comment by aliscott** [comment]
@b4nst thanks for reporting this. We will get a fix for this into v0.8.2.

**Comment by b4nst** [comment]
@aliscott thanks for the quick answer.

**Comment by b4nst** [comment]
Hum, after updating my configuration the issue disappear. I did not catch which part of the config were causing it though.

**Comment by aliscott** [comment]
@b4nst it looks like `ebs_options` is causing the issues if that's the part you changed.

**Comment by b4nst** [comment]
Indeed I changed it.

**Comment by b4nst** [comment]
That tool is dope btw, congrats.

**Comment by aliscott** [comment]
@b4nst thanks for reporting this! The fix has now been released in v0.8.2.
